ChopChop/Models/Recipe.swift

```diff
@@ -0,0 +1,23 @@
+import GRDB
+
+struct Recipe: Equatable {
+    var id: Int64?
+    var recipeCategoryId: Int64?
+    var name: String
+    var ingredients: [String: Quantity]
```

Comment:
This will need to be updated to work with `IngredientReference` in the future

Reply:
For now I wasn't very sure what the rest of you were going to use for your models, so I just used the simplest data structure, which can of course be modified. We just need to remember to update the mapping from the Record to the actual model.